nixos/modules/hardware/wooting.nix

```diff
@@ -1,12 +1,51 @@
 { config, lib, pkgs, ... }:
 
 with lib;
+let
+  cfg = config.hardware.wooting;
+  xinput-rules = pkgs.writeTextFile {
+    name = "71-wooting-xinput.rules";
+    text = ''
+      # Wooting One
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="3eb/ff01/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@03eb:ff01"
+
+      # Wooting Two
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="3eb/ff02/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@03eb:ff02"
+
+      # Wooting Two Lekker Edition
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="31e3/1210/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@31e3:1210"
+
+      # Wooting Two HE
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="31e3/1220/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@31e3:1220"
```

Comment:
This list seems to be outdated (just like [the official docs](https://help.wooting.io/en/article/guide-configuring-xinput-support-for-linux-69m32u)). For example, the 60HE is missing. It should be possible to construct the missing pieces with the identifiers from [here](https://help.wooting.io/en/article/wootility-configuring-device-access-for-wootility-under-linux-udev-rules-r6lb2o).